src/tests/spec/json/bson-corpus/timestamp.json

```diff
@@ -18,6 +18,11 @@
             "description": "Timestamp with high-order bit set on both seconds and increment",
             "canonical_bson": "10000000116100FFFFFFFFFFFFFFFF00",
             "canonical_extjson": "{\"a\" : {\"$timestamp\" : {\"t\" : 4294967295, \"i\" :  4294967295} } }"
+        },
+        {
+            "description": "Timestamp with high-order bit set on both seconds and increment (not UINT32_MAX)",
```

Comment:
This came in as part of the corpus test sync; it isn't related at all, but the driver seems to pass it without needing any changes